cloud/cmd/edgecontroller.go

```diff
@@ -0,0 +1,12 @@
+package main
```

Comment:
Since we will have edgecontroller, devicecontroller and cloudhub and so on in the cloud part.
Perhaps we can change this file name `edgecontroller.go` into something else. 
e.g. cloud-core like the edge_core
Thanks.

Reply:
:+1:

Reply:
It's a good idea @edisonxiang